manifests/plugin.pp

```diff
@@ -0,0 +1,26 @@
+#
+# == Define: pulp::plugin
+#
+# Install and configure a Pulp plugin
+#
+define pulp::plugin (
+  $package  = undef,
+) {
+
+  $real_package = $package ? {
+      undef       => "pulp-${name}-plugins",
+      default     => $package,
+  }
+
+  package { $real_package:
+    ensure => installed,
+  } ~>
+  exec {'migrate_pulp_db':
```

Comment:
Yes, sorry, this wasn't meant for review.  See my comment to @ehelms, #65 removes the `creates` restriction on the other resources, so we can just notify it once its merged.  #64 and #65 need to go first and then I'll refactor this.